Guide/tailwindcss.markdown

```diff
@@ -63,17 +63,15 @@ module.exports = {
         extend: {
         },
     },
-    purge: {
-        content: [
-            "../Web/View/**/*.hs",
-        ],
+    content: [
+        "../Web/View/**/*.hs",
+    ],
```

Comment:
Thanks. Let's change also the `safelist` - which needs to be outside of options. Also the comment is now outdated. It can be something like - https://github.com/Gizra/drupal-starter/blob/585c7d020b8adc27e5fd0e686df8c3241c89d094/web/themes/custom/server_theme/tailwind.config.js#L21-L24

Reply:
Done!